ksp_plugin/plugin.hpp

```diff
@@ -403,7 +415,11 @@ class Plugin {
           physics::KeplerianElements<Barycentric>> const& keplerian_elements,
       MassiveBody const& body);
 
+  // Whether |loaded_vessels_| contains |vessel|.
+  bool is_loaded(not_null<Vessel*> vessel) const;
+
   GUIDToOwnedVessel vessels_;
+  std::map<PartId, not_null<Vessel*>> part_id_to_vessel_;
```

Comment:
Document this map: "For each part, the vessel that this part belongs to.  The part is guaranteed to be part of the parts() map of the vessel, and owned by it."